terraform/aws-linux/main.tf

```diff
@@ -0,0 +1,137 @@
+variable "num_nodes" {
+	default = 1
+}
+
+variable "hostname_formatter" {
+	default = "amz-%d"
+}
+
+variable "ssh_user" {
+	default = "gila"
+}
+
+variable "ssh_key" {
+	default = "/home/gila/.ssh/id_rsa.pub"
```

Comment:
Similar to previous comment, `var.ssh_key`.

Reply:
thx fixed.